app/models/katello/concerns/subscription_aspect_host_extensions.rb

```diff
@@ -5,7 +5,18 @@ module SubscriptionAspectHostExtensions
       include ForemanTasks::Concerns::ActionTriggering
 
       included do
+        SUBSCRIPTION_STATUS_MAP = {
+          :valid => Katello::SubscriptionStatus::FULL_COMPLIANCE,
```

Comment:
I can never remember the nuances about subscription terminology.   @thomasmckay mind reviewing the status strings here?  (i'm more concerned about user facing strings that constant names).